Site/SiteAMQPModule.php

```diff
@@ -99,7 +99,7 @@ public function depends()
 	 * @param string $exchange   the job exchange name.
 	 * @param string $message    the job data.
 	 * @param array  $attributes optional. Additional message attributes. See
-	 *                           {@link http://www.php.net/manual/en/amqpexchange.publish.php}.
+	 *                           {@link https://secure.php.net/manual/fa/amqpexchange.publish.php}.
```

Comment:
English manual is missing these docs.

Reply:
I think the "Farsi" version has been taken down too. There's a "Danish" version here that currently works: http://docs.php.net/manual/da/amqpexchange.publish.php Do you know why they're taking these docs down?

Reply:
https://github.com/pdezwart/php-amqp/issues/17#issuecomment-91396642 🤦‍♂️

Reply:
Links fixed to point to https://github.com/pdezwart/php-amqp/blob/master/stubs/AMQPExchange.php#L155